pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -142,6 +142,34 @@ def __call__(self, value, system):
             restriction_on_landownership['Reference'] = reference
             restriction_on_landownership['Article'] = article
 
+        # One restriction entry per theme
+        theme_restriction = {}
+        text_element = [
+            'Information', 'Lawstatus_Code', 'Lawstatus_Text', 'ResponsibleOffice_Name',
+            'ResponsibleOffice_OfficeAtWeb', 'SymbolRef', 'TypeCode'
+        ]
+        for restriction_on_landownership in extract_dict.get('RealEstate_RestrictionOnLandownership', []):
+            theme = restriction_on_landownership['Theme_Code']
+            if theme not in theme_restriction:
+                theme_restriction[theme] = restriction_on_landownership
+                for element in text_element:
+                    restriction_on_landownership[element] = set([restriction_on_landownership[element]])
+                continue
+            curent = theme_restriction[theme]
+            # Number or array
+            for element in ['Area', 'Article', 'LegalProvisions', 'PartInPercent', 'Reference']:
```

Comment:
Area and PartInPercent may be None for linear and point restrictions.
In the case of linear restrictions, we should have a length value instead and nothing for a point restriction.

Reply:
Than what should I do for linear restrictions, Do we have an other property?

Reply:
Yes, if it's a linear restriction, we do have 'length' I'm not sure about the 'PartInPercent':
https://github.com/camptocamp/pyramid_oereb/blob/361e2e96973aa2c0c4d1cedb0dc61a5bcea4d148/pyramid_oereb/lib/renderer/extract/json_.py#L242-L267
:)

Reply:
Thanks, `Length` added :-)